packages/core/src/components/document-component/document-component.tsx

```diff
@@ -294,7 +294,7 @@ export class DocumentComponent implements ComponentInterface {
                     <span class="sr-only">{this.open ? "Invouwen" : "Uitvouwen"}</span>
                   </button>
                 )}
-                <div id="heading-title">
+                <span id="heading-title">
```

Comment:
Tijdens het ontwikkelen zag ik dat er een `<div>` in de heading zat. Opgelost met deze `span` en `display: block;`.